src/api.ts

```diff
@@ -144,18 +169,19 @@ export class MirrordAPI {
   }
 
   // Return environment for the spawned mirrord cli processes.
-  private static getEnv(): NodeJS.ProcessEnv {
+  private static getEnv(configEnv: EnvVars): NodeJS.ProcessEnv {
     // clone env vars and add MIRRORD_PROGRESS_MODE
     return {
       // eslint-disable-next-line @typescript-eslint/naming-convention
       "MIRRORD_PROGRESS_MODE": "json",
+      ...configEnv,
       ...process.env,
```

Comment:
```suggestion
      ...process.env,
      ...configEnv,
      // eslint-disable-next-line @typescript-eslint/naming-convention
      "MIRRORD_PROGRESS_MODE": "json",
```

`...` operator will overwrite previous values, I think we want this order